### PR TITLE
code tidy

### PR DESCRIPTION
**Pull request by moloko**

### CODE REVIEW

**Comment by joannecas** [comment]
+1 
looks and works fine

File: js/completionCalculations.js [hunk 1]
```diff
@@ -45,8 +45,8 @@ define([
                 "assessmentTotal": assessmentComponentsTotal
             };
 
-            if (contentObjectModel.get("_pageLevelProgress") && contentObjectModel.get("_pageLevelProgress")._showPageCompletion !== false 
-                && Adapt.course.get("_pageLevelProgress") && Adapt.course.get("_pageLevelProgress")._showPageCompletion !== false) {
+            if (contentObjectModel.get("_pageLevelProgress") && contentObjectModel.get("_pageLevelProgress")._showPageCompletion !== false &&
```
Comment (lc-thomasberger) [inline js/completionCalculations.js]: split up into variables?
```
var showPageCompletionOnCO = contentObjectModel.get("_pageLevelProgress") && contentObjectModel.get("_pageLevelProgress")._showPageCompletion !== false;
var showPageCompletionOnCourse = Adapt.course.get("_pageLevelProgress") && Adapt.course.get("_pageLevelProgress")._showPageCompletion !== false;

if (showPageCompletionOnCO && showPageCompletionOnCourse) {
  // ...
}

```
not tested the logic and not sure about the var names, but it makes it easier to read

Comment (moloko) [reply js/completionCalculations.js]: I'll have a look in a bit..!

**Comment by moloko** [comment]
@oliverfoster could you check 4b13d1d and make sure that's OK still?

File: js/PageLevelProgressMenuView.js [hunk 2]
```diff
@@ -37,16 +28,16 @@ define(function(require) {
         },
 
         updateProgressBar: function() {
+            var percentageOfCompleteComponents;
```
Comment (tomgreenfield) [inline js/PageLevelProgressMenuView.js]:
```js
var percentageComplete = this.model.get('completedChildrenAsPercentage') || 0;
```
❓

File: js/PageLevelProgressNavigationView.js [hunk 3]
```diff
@@ -1,47 +1,44 @@
...
-                this.updateProgressBar();
-            }, this));
-        },
-
-        events: {
-            'click': 'onProgressClicked'
+            _.defer(this.updateProgressBar.bind(this));
         },
-
+
```
Comment (tomgreenfield) [inline js/PageLevelProgressNavigationView.js]: Random whitespace.

File: js/PageLevelProgressMenuView.js [hunk 4]
```diff
@@ -28,12 +28,7 @@ define([
...
-            var percentageOfCompleteComponents;
-            if (this.model.get('completedChildrenAsPercentage')) {
-                percentageOfCompleteComponents = this.model.get('completedChildrenAsPercentage');
-            } else {
-                percentageOfCompleteComponents = 0;
-            }
+            var percentageComplete = this.model.get('completedChildrenAsPercentage') || 0;
 
             // Add percentage of completed components as an aria label attribute
             this.$('.page-level-progress-menu-item-indicator-bar .aria-label').html(this.ariaText + Math.floor(percentageOfCompleteComponents) + '%');
```
Comment (tomgreenfield) [inline js/PageLevelProgressMenuView.js]: Sorry – because I made the variable name more concise in my suggestion, it also needs updating here.

**Pull request merged**